### ISSUE TITLE
[BUG] Unable to load lap data for Saudi Arabia race

### ISSUE THREAD

**Issue by bambz96** [theOehrly/Fast-F1#135]
### Describe the issue:

Lap data for the 2022 Saudi Arabia GP fails to load. I did some debugging and it appears to be related to the section of code that creates the total laps counter in `_load_laps_data`. That section executes fine for all drivers except Tsunoda (driver 22). I have not been able to figure out a workaround.

### Reproduce the code example:

```python
import fastf1 as ff1

session = ff1.get_session(2022, 2, 'R')
session.load()
laps = session.laps
```

### Error message:

```shell
core           INFO     Loading data for Saudi Arabian Grand Prix - Race [v2.2.1]
api            INFO     Fetching driver list...
api            INFO     Fetching timing data...
api            INFO     Parsing timing data...
api            INFO     Fetching timing app data...
core           INFO     Processing timing data...
api            INFO     Fetching session status data...
core        WARNING     Failed to load lap data!
api            INFO     Fetching car data...
api            INFO     Parsing car data...
api            INFO     Fetching position data...
api            INFO     Parsing position data...
api         WARNING     Driver 241: Position data is incomplete!
api         WARNING     Driver 242: Position data is incomplete!
core        WARNING     Failed to load telemetry data!
api            INFO     Fetching weather data...
core           INFO     Finished loading data for 19 drivers: ['1', '16', '55', '11', '63', '31', '4', '10', '20', '44', '24', '27', '18', '23', '77', '14', '3', '6', '22']
Traceback (most recent call last):
 [REMOVED PRECEEDING TRACE]
  File "[REMOVED]\saudigpissue.py", line 5, in <module>
    laps = session.laps
  File "[REMOVED]\fastf1\core.py", line 945, in laps
    return self._get_property_warn_not_loaded('_laps')
  File "[REMOVED]\fastf1\core.py", line 910, in _get_property_warn_not_loaded
    raise DataNotLoadedError("The data you are trying to access has not "
fastf1.core.DataNotLoadedError: The data you are trying to access has not been loaded yet. See `Session.load`
```

**Comment by theOehrly** [comment]
That whole section of code is a bit of trickery, and a fake lap for Yuki is added even though it should not be added. Commenting it out is fine for this race.

@ezkomoly can you clarify, what do you mean, you can't use the 'LapNumber' attribute at all? I can't reproduce this right now.

**Comment by ezkomoly** [comment]
> That whole section of code is a bit of trickery, and a fake lap for Yuki is added even though it should not be added. Commenting it out is fine for this race.
> 
> @ezkomoly can you clarify, what do you mean, you can't use the 'LapNumber' attribute at all? I can't reproduce this right now.

I checked again, and it was actually my bad. I didn't look at the error message. 
I had plt.savefig trying to save to my external SSD while it was not connected.
So not adding the "fake" laps to Yuki does work and 
`.loc[laps['LapNumber'] == 49.iloc[0]`
does plot the data from lap 49.

**Comment by theOehrly** [comment]
Fixed now in new version v2.2.2
